### ISSUE TITLE
Boss health isn't changing

### ISSUE THREAD

**Issue by Dhaloi** [CharonXXIII/Pixel-Worlds-C-Cheat#1]
ok so, i was trying this hacks and noticed health of the "boss" isnt changing at all,

**Comment by CharonXXIII** [comment]
Yeah, the reason is that I'm just asking for the boss health while I need to ask for the boss health for that one and for it to update, I'll be fixed maybe if I'm bothered about it and if I continue updating the public verison.
